root/static/scripts/common/utility/formatSetlist.js

```diff
@@ -37,6 +37,9 @@ const linkRegExp =
  * https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references
  */
 function decodeSomeHTMLEntities(content: string) {
+  if (!content) {
+    return '';
+  }
```

Comment:
Even empty strings should have the replace method, so I guess the parameter type of `content: string` is also wrong and can be fixed?

Reply:
Yeah, it was `undefined`, good point.